docs/screendesigndraft.md

```diff
@@ -0,0 +1,50 @@
+# SignMeUp GUI设计方案
```

Comment:
This file should be part of the `design.md`.

Reply:
I will integrate this later, thanks.